### PR TITLE
simple example test using version

### PR DESCRIPTION
**Pull request by BradfordMedeiros**
Added a simple example test for full integration testing for the bundlectl command.
I added a simple, simple test which simply verifies that the version command exists. 

So just for clarity, and maybe this should be documented since it's non-obvious, but these sh_tests basically pass or fail depending on the exit code.

I plan on adding in a variety of tests today, but I just want to get this in asap as an example test.

### CODE REVIEW

**Comment by BradfordMedeiros** [comment]
Also one thought @Kashomon --> would you be opposed to me adding a top level makefile? 
I personally like to standardize on make build, make test, make publish, make install, etc 

I'm using bazel as a dependency to test.  I think that's okay. 
Alternatively we could do this via a make script.

In any case, the test code is thankfully not too specific to bazel.

**Comment by Kashomon** [comment]
I'd prefer to use Bazel over make, since that has nice integration with Prow. Also, bazel has other benefits too (blaze query, for example) that are nice.

Did you see https://github.com/GoogleCloudPlatform/k8s-cluster-bundle/pull/242? Is this a response to that CL?

**Comment by Kashomon** [comment]
If we're going to do Shell, I would prefer we follow the GBash style guide https://google.github.io/styleguide/shell.xml and probably also use the GBash test runner

**Comment by Kashomon** [comment]
oops. didn't mean to close

**Comment by BradfordMedeiros** [comment]
@Kashomon no this was miscoordinated if anything, not supposed to be in response to the other pr.

**Comment by Kashomon** [comment]
No worries

File: cmd/bundlectl/integration-tests/version.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
+bundlectl="./$1"
```
Comment (Kashomon) [inline cmd/bundlectl/integration-tests/version.sh]: + #!/bin/bash
+ copyright header 

set -o errexit
set -o nounset
set -o pipefail

File: cmd/bundlectl/integration-tests/BUILD [hunk 2]
```diff
@@ -0,0 +1,9 @@
+sh_test(
```
Comment (Kashomon) [inline cmd/bundlectl/integration-tests/BUILD]: Can we use the directory `integration_tests` or `integration`? `blaze` (internal) doesn't play nice with foo-bar directories, as I recall

File: cmd/bundlectl/integration-tests/version.sh (hunk 1 above)
Comment (Kashomon) [reply cmd/bundlectl/integration-tests/version.sh]: this is specifically what I meant about following the bash style guide.

**Comment by BradfordMedeiros** [comment]
@Kashomon is this okay to merge?  I think it's valuable to have full E2E tests for cmd line utiltiies.

**Comment by Kashomon** [comment]
/lgtm
Sure. I would strongly prefer most of our integration tests be written in Go using the thing I submitted, but it's useful to have a couple smoke tests like this. Doesn't need to be very many (maybe version is enough?)

**Comment by BradfordMedeiros** [comment]
@Kashomon my intent is to have full end to end tests.  Basically move the example directory into tests. 
Consider that most of these tests will be having input files, and comparing them to a resultant set of output files.  Bash is excellent for that.

**Comment by Kashomon** [comment]
I don't think shell tests give more coverage than the go integration tests

**Pull request merged**